helm/hpcc/templates/eclagent.yaml

```diff
@@ -78,6 +79,7 @@ data:
             accessDali: "yes"
             accessEsp: "yes"
         spec:
+          {{- include "hpcc.placementMatchName" (dict "root" $ "pod" (printf "%s-" $apptype) "group" "eclagent") | indent 10 }}
```

Comment:
It would be good to have name of agent that this belongs to here, e.g. (printf %s-%s .name $apptype) (the .name could be changed instead to reflect that ...
So that where there multiple eclagent targets, they could have different placements.